XRTK.Oculus/Packages/com.xrtk.oculus/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "com.xrtk.oculus",
   "displayName": "XRTK.Oculus",
   "description": "The Oculus platform components for the Mixed Reality Toolkit",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "unity": "2019.1",
   "license": "MIT",
   "repository": {
```

Comment:
Merge in the changes from #14 or update the package.json

Reply:
Done